src/models/db/notification.js

```diff
@@ -1,18 +0,0 @@
-'use strict'
-
-const Model = require('five-bells-shared').Model
-const PersistentModelMixin = require('five-bells-shared').PersistentKnexModelMixin
-const knex = require('../../lib/knex').knex
-
-class Notification extends Model {
```

Comment:
This already doesn't exist on master, this pull request is based on a very old version of the code...